### PR TITLE
add remove command

### PR DESCRIPTION
**Pull request by triangular2357**

### CODE REVIEW

File: cogs/suggest.py [hunk 1]
```diff
@@ -280,7 +288,30 @@ async def deny(self, ctx, id: str = '999999999', *, reason: str = ''):
...
+    @commands.command()
+    async def remove(self, ctx,* ,modId: str = ''):
+        if not modId:
+            return
+
+        if self.data == None:
+            self.load_storage()
+
+        modId=int(modId.strip())
+
```
Comment (NickMesser) [inline cogs/suggest.py:309]: Lets check if Mod exists before we do anything else.

File: cogs/suggest.py [hunk 2]
```diff
@@ -280,7 +288,30 @@ async def deny(self, ctx, id: str = '999999999', *, reason: str = ''):
...
+        for x in self.data['Suggestions']:
+            if x['ModId'] == modId:
+                messageId = x['MessageId']
+                oldChannel = self.bot.get_channel(x['ChannelId'])
+                oldMsg = await oldChannel.fetch_message(messageId)
+                await oldMsg.delete()
+
+        self.remove_mod_from_database(modId)
```
Comment (NickMesser) [inline cogs/suggest.py]: Lets make this a bool so we can check for a true responce.

File: cogs/suggest.py [hunk 3]
```diff
@@ -77,11 +77,14 @@ def add_mod(self, mod, message, comments, stage : str = 'suggestion'):
 
     def remove_mod_from_database(self, modId=0):
         if not modId:
-            return
+            return False
```
Comment (NickMesser) [inline cogs/suggest.py:80]: No need to call it here again.

**Review by NickMesser — CHANGES_REQUESTED** [review]

File: cogs/suggest.py [hunk 4]
```diff
@@ -77,11 +77,14 @@ def add_mod(self, mod, message, comments, stage : str = 'suggestion'):
...
+        if not self.mod_exists(modId):
```
Comment (NickMesser) [inline cogs/suggest.py]: No need to call this again.

**Review by NickMesser — CHANGES_REQUESTED** [review]

**Comment by triangular2357** [comment]
I think now every requested thing is added if not let me know

**Pull request merged**